server/server.js

```diff
@@ -1,29 +1,29 @@
 const path = require('path');
 const express = require('express');
+const testsRoutes = require('./routes/tests');
 
 const app = express();
+const PORT = 3000;
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
-
-const PORT = 3000;
+app.use('/api/tests', testsRoutes);
+app.use('/dist', express.static(path.join(__dirname, '../dist')));
 
 app.get('/', (req, res) =>
   res.status(200).sendFile(path.join(__dirname, '../client/index.html'))
-),
-
-app.use('/dist', express.static(path.join(__dirname, '../dist')));
```

Comment:
I had this because I need to serve the logo picture to the client.  Could you help achieve that?

Reply:
This was moved to line 11 to bundle the app.uses